### ISSUE TITLE
VRP application does not compile when using VPATH configuration

### ISSUE THREAD

**Issue by svigerske** [coin-or/SYMPHONY#81]
Issue created by migration from Trac.

Original creator: fmargot

Original creation time: 2008-04-21 13:12:50

Assignee: menal

Version: 

CC:  fmargot@andrew.cmu.edu

When using a VPATH configuration in directory SYMPHONY-5.1.7/build:

../configure -C

make

make install

Then going into SYMPHONY-5.1.7/build/SYMPHONY/Applications/VRP and typing 

make

I get:

VRP>make

make: *** No rule to make target `src/Master/vrp_main.o', needed by `vrp'.  Stop

**Comment by svigerske** [comment]
Comment by @tkralphs created at 2008-04-21 15:17:08

It works fine for me. I think the problem is probably that you need to configure with "--with-application," i.e., 

../configure -C --with-application

Try that and open this ticket again if it doesn't work.

**Comment by svigerske** [comment]
Comment by @tkralphs created at 2008-04-21 15:17:08

Resolution: worksforme

**Comment by svigerske** [comment]
Comment by fmargot created at 2008-04-21 18:10:27

configuring in SYMPHONY-5.1.7/build with 

../configure --with-application

and then typing 

make

in SYMPHONY-5.1.7/build/SYMPHONY/Applications/VRP

does not work. I still get:

VRP>make

make: *** No rule to make target `src/Master/vrp_main.o', needed by `vrp'.  Stop.

**Comment by svigerske** [comment]
Comment by fmargot created at 2008-04-21 18:10:27

Resolution changed from worksforme to

**Comment by svigerske** [comment]
Comment by fmargot created at 2008-04-21 18:10:27

Changing status from closed to reopened.

**Comment by svigerske** [comment]
Comment by @tkralphs created at 2008-04-21 18:27:15

It looks like maybe you didn't "make" in SYMPHONY-5.1.7/build/ first. You need to do "make" and "make install" for SYMPGHONY itself first. Is that maybe the problem?

**Comment by svigerske** [comment]
Comment by @tkralphs created at 2008-04-21 20:37:07

OK, I have reproduced this now. Actually, the problem is that in 5.1.7, you had to do make in SYMPHONY-5.1.7/SYMPHONY/Applications/VRP, not in SYMPHONY-5.1.7/build/SYMPHONY/Applications/VRP. However, we have recently changed this so that it works as you expected it to. The fix is in stable/5.1 and will be in 5.1.8, which will come out in a day or two. For now, use stable/5.1.
